config/manager/manager.yaml

```diff
@@ -63,6 +63,9 @@ spec:
         args:
           - --leader-elect
           - --health-probe-bind-address=:8081
+          - --grpc-insecure
+          - --grpc-token-file=/var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
Should this (and the value below) be part of a configuration file or something? It does feel a little strange to have this hardcoded for our specific testing environment.

Reply:
Yeah, this should be loaded from environment variables like we're doing with other configuration options (e.g., `CLOUDKIT_CLUSTER_CREATE_WEBHOOK`). I'll update the PR.